clang/lib/CodeGen/BackendUtil.cpp

```diff
@@ -599,6 +599,19 @@ void EmitAssemblyHelper::CreatePasses(legacy::PassManager &MPM,
          CodeGenOpts.PrepareForThinLTO));
   }
 
+  // FIXME: This code is a workaround for a number of problems with optimized
+  // SYCL code for SPIR target. This change trying to balance between doing too
```

Comment:
for "the SPIR target" or "for SPIR targets".

Reply:
Thanks. Applied all suggestions.